writer.go

```diff
@@ -67,7 +67,7 @@ func (c *Client) NewBufferingWriter(bucket string, org string, flushInterval tim
 	if onError == nil {
 		onError = func(_ error) {}
 	}
-	w := &LPWriter{c: c, buf: switchableBuffer{&bytes.Buffer{}}, flushSize: flushSize, flushInterval: flushInterval, stop: make(chan struct{}), onError: onError}
+	w := &LPWriter{c: c, buf: switchableBuffer{&bytes.Buffer{}}, flushSize: flushSize, flushInterval: flushInterval, stop: make(chan struct{}), onError: onError, bucket: bucket,org: org}
```

Comment:
I think you need a `gofmt` here. Perhaps worth adding that to the circleci check as well.